Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAuditdPlugin.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python
+#============================================================================
+# Copyright (C) Microsoft Corporation, All rights reserved.
+#============================================================================
+
+import os
+import sys
+import imp
+import codecs
+import subprocess
+
+protocol = imp.load_source('protocol', '../protocol.py')
+nxDSCLog = imp.load_source('nxDSCLog', '../nxDSCLog.py')
+
+LG = nxDSCLog.DSCLog
+
+PLUGIN_CONF = '/etc/audisp/plugins.d/auoms.conf'
+OMSAGENT_CONF = '/etc/opt/microsoft/omsagent/conf/omsagent.d/auoms.conf'
+MODULE_RESOURCE_DIR = '/opt/microsoft/omsconfig/modules/nxOMSAuditdPlugin/DSCResources/MSFT_nxOMSAuditdPluginResource'
```

Comment:
Can this resource path be discovered dynamically instead of hardcoded path?  I wonder if with publishing a new resource , change in the resource version causes this path being different.